### PR TITLE
Added a cpjobqueue deployment and redis deployment

### PR DESCRIPTION
**Pull request by Pchelolo**
Added CPJobQueue deployment to the mediawiki-dev kubernetes config.

### CODE REVIEW

File: mediawiki-dev.yaml [hunk 1]
```diff
@@ -418,11 +447,39 @@ spec:
...
             value: http://mediawiki-svc
           - name: RESTBASE_URL
             value: http://restbase-svc:7231
           - name: KAFKA_BROKER_LIST
             value: kafka-svc:9092
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: change-propagation-deploy
```
Comment (d00rman) [inline mediawiki-dev.yaml]: Did you mean(TM) cp-jobqueue-deploy ?

File: mwctl/Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
+export GOPATH=$(CURDIR)/.build
```
Comment (d00rman) [inline mwctl/Makefile]: Why adding the mwctl stuff here? Haven't we got [wikimedia/mwctl](https://github.com/wikimedia/mwctl) for that?

**Comment by Pchelolo** [comment]
@d00rman updated. All the unnecessary pieces were due to a lack of a rebase.

**Comment by d00rman** [comment]
@Pchelolo mwctl stuff is still there?

**Comment by Pchelolo** [comment]
> @Pchelolo mwctl stuff is still there?

Rebasing magic. Fixed.

**Pull request merged**